templates/data-sources/elasticsearch_security_role.md.tmpl

```diff
@@ -0,0 +1,17 @@
+---
+subcategory: "Security"
+layout: ""
+page_title: "Elasticstack: elasticstack_elasticsearch_security_role Data Source"
+description: |-
+  Retrieves roles in the native realm.
+---
+
+# Data Source: elasticstack_elasticsearch_security_role
+
+Use this data source to  information about existing Elasticsearch role. See, https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-role.html
```

Comment:
```suggestion
Use this data source to get information about an existing Elasticsearch role. See, https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-get-role.html
```

Maybe there's other words you intended to have here, happy to use those instead too :)

Reply:
Whoops, I guess I just deleted it by mistake, fixed🙏
https://github.com/elastic/terraform-provider-elasticstack/pull/177/commits/6ec61a462cd1f681df1ff47ae1208775ceadbc5d